Add EMnify integration (#1534)
DataDog/integrations-extras: emnify/README.md

## Overview
[EMnify][1] is a cellular IoT connectivity platform that keeps devices
connected and secure.

Use the Datadog-EMnify integration to collect metrics and data usage from IoT EMnify devices.

## Setup
Follow the [Emnify integration guide][2] to configure streaming of usage data, using the **Integration steps** and **Verifying the integration** sections.

### Monitors

Everybody has different usage patterns, so in order for monitors to reflect your specific case you must
define workload-based boundaries and sensitivity.
For more information read the [Forecast monitor][4] and [Anomaly monitor][5] documentation.

### Dashboard

> You can pick a period in [dashboard][6] to filter data displayed.

Reviewer: ```suggestion
Choose a time frame in the [dashboard][6] to filter the data displayed.
```

## Overview

> [EMnify][1] cellular IoT connectivity platform keeps your devices
> connected and secure wherever you deploy. Our network-agnostic SIMs
> let your applications connect to more than 540 networks in over 180 countries.
> We also offer the largest LTE-M coverage on the market.

Reviewer: ```suggestion
[EMnify][1] is a cellular IoT connectivity platform that keeps devices
connected and secure.
```

Use the Datadog-EMnify integration to collect metrics and data usage from IoT EMnify devices.

## Setup

> Configure streaming of Usage Data following the [integration guide][2],
> sections "Integration Steps" and "Verifying the integration".

Reviewer: ```suggestion
Follow the [integration guide][2] to configure streaming of usage data, using the **Integration steps** and **Verifying the integration** sections.
sections.
```

### Monitors

Everybody has different usage patterns so in order for monitors to reflect your specific case you must
define workload-based boundaries and sensitivity.

> For more information check [Forecast monitor docs][4] and [Anomaly monitor docs][5].

Reviewer: ```suggestion
For more information read the [Forecast monitor][4] and [Anomaly monitor][5] documentation.
```

### Dashboard

> You can pick a period to filter data displayed.

Reviewer: Is there a link to the dashboard that could be provided here? Such as [this example](https://github.com/DataDog/integrations-extras/blob/4c8f758c7e9afd4680043600b278be2d5a325f93/datazoom/README.md#dashboard).

Author: Great advice thanks!
